### ISSUE TITLE
enhancement suggestion: integrate "reports to" user field into Workflow actions

### ISSUE THREAD

**Issue by EZS-JD** [salesagility/SuiteCRM#878]
I'd like to see the possibility to choose the "reports to" user in workflows, e.g. as recipient of emails, or to assign a record to the superior or create a note to the superior, etc...

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [enhancement suggestion: integrate "reports to" user field into Workflow actions](https://trello.com/c/FzD2Q6jF/121-enhancement-suggestion-integrate-reports-to-user-field-into-workflow-actions)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
